src/transformers/models/omdet_turbo/configuration_omdet_turbo.py

```diff
@@ -0,0 +1,311 @@
+        backbone_features_only (`bool`, *optional*, defaults to `True`):
+            Whether to output only the features of the vision backbone (no head built on top).
+        use_pretrained_backbone (`bool`, *optional*, defaults to `True`):
+            Whether to use a pretrained timm vision backbone.
+        backbone_image_size (`int`, *optional*, defaults to 640):
+            The image size for the vision backbone.
+        encoder_hidden_dim (`int`, *optional*, defaults to 256):
+            The hidden dimension of the encoder.
+        decoder_hidden_dim (`int`, *optional*, defaults to 256):
+            The hidden dimension of the decoder.
+        backbone_feat_channels (`tuple(int)`, *optional*, defaults to `[256, 256, 256]`):
+            The projected vision features channels used as inputs for the decoder.
+        num_feature_levels (`int`, *optional*, defaults to 3):
+            The number of feature levels for the multi-scale deformable attention module of the decoder.
+        disable_custom_kernels (`bool`, *optional*, defaults to `False`):
+            Whether to disable custom kernels.
+        text_projection_in_features (`int`, *optional*, defaults to 512):
+            The input features for the text projection.
+        text_projection_out_features (`int`, *optional*, defaults to 512):
+            The output features for the text projection.
+        num_queries (`int`, *optional*, defaults to 900):
+            The number of queries.
+        layer_norm_eps (`float`, *optional*, defaults to 1e-05):
+            The epsilon value for layer normalization.
+        batch_norm_eps (`float`, *optional*, defaults to 1e-05):
+            The epsilon value for batch normalization.
+        csp_activation (`str`, *optional*, defaults to `"silu"`):
+            The activation function of the Cross Stage Partial (CSP) networks of the encoder.
+        conv_norm_activation (`str`, *optional*, defaults to `"gelu"`):
+            The activation function of the ConvNormLayer layers of the encoder.
+        ffn_encoder_activation (`str`, *optional*, defaults to `"relu"`):
+            The activation function for the feedforward network of the encoder.
+        hidden_expansion (`int`, *optional*, defaults to 1):
+            The hidden expansion of the CSP networks.
+        dropout (`float`, *optional*, defaults to 0.0):
+            The dropout rate of the encoder.
+        activation_dropout (`float`, *optional*, defaults to 0.0):
+            The activation dropout rate of the encoder ffn.
+        encoder_in_channels (`list(int)`, *optional*, defaults to `[192, 384, 768]`):
+            The input channels for the encoder.
+        encoder_feat_strides (`list(int)`, *optional*, defaults to `[8, 16, 32]`):
+            The feature strides for the encoder.
+        encode_proj_layers (`list(int)`, *optional*, defaults to `[2]`):
+            The projection layers for the encoder.
```

Comment:
Does this need to be a list?

Reply:
This corresponds to the indices of the input features projected by each layers (I updated the name and the docstrings), here there is only one layers but we do use both the length and the value of this list